src/js/select2/selection/multiple.js

```diff
@@ -26,73 +20,78 @@ define([
 
     MultipleSelection.__super__.bind.apply(this, arguments);
 
-    var id = container.id + '-container';
-    this.$selection.find('.select2-selection__rendered').attr('id', id);
+    var id = container.id + "-container";
+    this.selection
+      .querySelector(".select2-selection__rendered")
+      .setAttribute("id", id);
 
-    this.$selection.on('click', function (evt) {
-      self.trigger('toggle', {
-        originalEvent: evt
+    this.selection.addEventListener("click", function (evt) {
+      self.trigger("toggle", {
```

Comment:
I think it might be worth checking this code, this `.trigger` is from JQuery. Probably here an event should be raised.